test/unit/views/StatementPageSpec.scala

```diff
@@ -0,0 +1,125 @@
+  implicit val fakeRequest: FakeRequest[_] = FakeRequest()
+  implicit val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
+
+  val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
+  implicit val messages: Messages = messagesApi.preferred(fakeRequest)
+
+  private val fullyAccessibleServiceStatement = AccessibilityStatement(
+    serviceKey = "fully-accessible-service",
+    serviceName = "fully accessible service name",
+    serviceHeaderName = "Fully Accessible Name",
+    serviceDescription = "Fully accessible description.",
+    serviceDomain = "www.tax.service.gov.uk/test/",
+    serviceUrl = "fully-accessible",
+    contactFrontendServiceUrl = "some.contact-frontend",
+    complianceStatus = FullCompliance,
+    accessibilityProblems = Seq(),
+    milestones = Seq(),
+    accessibilitySupportEmail = None,
+    accessibilitySupportPhone = None,
+    serviceSendsOutboundMessages = false,
+    serviceLastTestedDate = new GregorianCalendar(2020, Calendar.FEBRUARY, 28).getTime,
+    statementCreatedDate = new GregorianCalendar(2020, Calendar.MARCH, 15).getTime,
+    statementLastUpdatedDate = new GregorianCalendar(2020, Calendar.MAY, 1).getTime
+  )
+
+  "Given an Accessibility Statement for a fully accessible service, rendering a Statement Page" should {
+    "return HTML containing the header containing the service name" in {
+      val statementPage = app.injector.instanceOf[StatementPage]
+      val statementPageHtml = statementPage(fullyAccessibleServiceStatement)
+      statementPageHtml.toString() should include("""<h1 class="govuk-heading-xl">Accessibility statement for fully accessible service name</h1>""")
+    }
+
+    "return HTML containing the expected introduction with link to the service" in {
+      val statementPage = app.injector.instanceOf[StatementPage]
+      val statementPageHtml = statementPage(fullyAccessibleServiceStatement)
+      statementPageHtml.toString() should include("""<a class="govuk-link" href="www.tax.service.gov.uk/test/">www.tax.service.gov.uk/test/</a>.""")
+    }
+
+    "return HTML containing the expected using service information with service description" in {
+      val statementPage = app.injector.instanceOf[StatementPage]
+      val statementPageHtml = statementPage(fullyAccessibleServiceStatement)
+      statementPageHtml.toString() should include("""<p class="govuk-body">Fully accessible description.</p>""")
+    }
+
+    "return HTML containing the expected accessibility information stating that the service is fully compliant" in {
+      val statementPage = app.injector.instanceOf[StatementPage]
+      val statementPageHtml = statementPage(fullyAccessibleServiceStatement)
+      statementPageHtml.toString() should include("""<p class="govuk-body">This service is fully compliant with the<a class="govuk-link" href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1 AA standard</a></p>""")
```

Comment:
We've used the contentAsString helper previously rather than toString(), not actually sure if one is preferred over the other.

Reply:
Good suggestion. Change made and pushed up to the branch.